### DISCUSSION TITLE
multiple lights in a room

### DISCUSSION THREAD

**Discussion by domemshur** [dwainscheeren/dwains-lovelace-dashboard#436] [General]
How do i put multiple lights in a room? I only can put one.

**Comment by confucius78** [comment]
Make a light group and add that to the config

**Reply by domemshur** [reply]
Can you only have one light/group per room?

**Reply by diothar** [reply]
Dwains 3.0 will has much better management of multiple lightbulbs. It’s in beta.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
